FRINXio/frinx-frontend
# Add coordinates from arango

Replace old mutation that used postgres db as a source for position with new mutation that uses the topology api as a source.

HOW TO TEST:
- open topology
- move nodes around
- refresh page, their position should be saved
